mouse.s

```diff
@@ -75,6 +75,10 @@ WGEnableMouse:
 
 	SETSWITCH PAGE2OFF
 
+	sec
+	.byte $C2, $01 ;REP #$01	; clear carry flag on 65816
```

Comment:
We're relying on the fact that the 65C02 treats undefined opcodes as NOPs here, so we should document that obscure fact in the comment. Someone may wish to support the NMOS 6502 someday, and could be burned by this ($C2 is suspected to halt some models of NMOS 6502).